planning/behavior_path_planner/autoware_behavior_path_planner_common/include/autoware/behavior_path_planner_common/utils/path_shifter/path_shifter.hpp

```diff
@@ -202,17 +202,20 @@ class PathShifter
   /**
    * @brief Check if the shift points are aligned in order and have no conflict range.
    */
-  bool checkShiftLinesAlignment(const ShiftLineArray & shift_lines) const;
+  bool check_shift_lines_alignment(const ShiftLineArray & shift_lines) const;
 
-  static void addLateralOffsetOnIndexPoint(ShiftedPath * path, double offset, size_t index);
+  static void add_lateral_offset_on_index_point(ShiftedPath * path, double offset, size_t index);
 
-  static void shiftBaseLength(ShiftedPath * path, double offset);
+  static void shift_base_length(ShiftedPath * path, double offset);
```

Comment:
@go-sakayori Could you add doxygen comment?

Reply:
Fixed in 878e09071e003c27e64ccaf92735eb4bc3c76096